analyzers/rspec/cs/S1117_c#.json

```diff
@@ -13,5 +13,7 @@
   "defaultSeverity": "Major",
   "ruleSpecification": "RSPEC-1117",
   "sqKey": "S1117",
-  "scope": "All"
+  "scope": "All",
+  "securityStandards": {
+  }
```

Comment:
Would be great to have the tool updated to skip the generation of empty sections.

Reply:
That was probably just me, with manual edits.

Reply:
Ah, this appears in a lot of places.

Reply:
I just rerun that 6 LOC script I used to remove the CERT :)